pkg/injector/test_fixtures/expected_envoy_bootstrap_config.yaml

```diff
@@ -72,7 +71,6 @@ static_resources:
     name: liveness_cluster
     type: STATIC
   - connect_timeout: 1s
-    lb_policy: ROUND_ROBIN
```

Comment:
We should be setting this in the XDS struct.

Reply:
Round robin is the default lb_policy. I am currently setting the lb_policy 'LbPolicy: xds_cluster.Cluster_ROUND_ROBIN', but the value of 'xds_cluster.Cluster_ROUND_ROBIN' is 0. The protojson marshaler considers 0 an empty value and it is therefore omitted. I have not found a way to keep default values. I have the same issue with setting 'codec_type: AUTO' for a filter.

Reply:
Here is the issue discussing protojson support for masking what fields are omitted https://github.com/golang/protobuf/issues/1183